src/main/java/org/spongepowered/api/scoreboard/critieria/Criteria.java

```diff
@@ -74,6 +74,43 @@
      */
     public static final Criterion TRIGGER = DummyObjectProvider.createFor(Criterion.class, "TRIGGER");
 
+    /**
+     * Represents a {@link Criterion} which causes an {@link Objective}
+     * to have a score that represents the amount of hunger points the
+     * player has, on a scale of 0-20. May appear as 0 for players before
+     * their foodLevel has changed for the first time.
+     */
+    public static final Criterion FOOD = DummyObjectProvider.createFor(Criterion.class, "FOOD");
```

Comment:
Please put these fields in alphabetical order in this class (there is a grade task to do it but I cannot for the life of me remember what it is).

Reply:
`./gradlew sortClassFields`

iirc

Reply:
That said, if there is a line break on one of the lines defining a catalog, DO NOT RUN THAT COMMAND. It will not end well for the `CatalogTypes` class.

Reply:
sortClassFields 👍